### PR TITLE
Added travis, updated gradle dependencies and fixed ns errors

### PR DESCRIPTION
**Pull request by KartikShankhavaram**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -2,8 +2,9 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/LNMIIT_Android_App.iml" filepath="$PROJECT_DIR$/LNMIIT_Android_App.iml" />
+      <module fileurl="file://$PROJECT_DIR$/LNMIIT-Official-Android-App.iml" filepath="$PROJECT_DIR$/LNMIIT-Official-Android-App.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```
Comment (saketkumar) [inline .idea/modules.xml]: Remove this changes!

**Review by saketkumar — CHANGES_REQUESTED** [review]
LGTM! 
Do the changes and I'll merge this.

File: .idea/modules.xml (hunk 1 above)
Comment (KartikShankhavaram) [reply .idea/modules.xml]: Removed!

**Comment by KartikShankhavaram** [comment]
Please consider the other PR I have sent.

**Pull request closed without merging**